website/static/js/wikiTreebeard.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
```

Comment:
It might make sense to rename this file to `wikiSettingsTreebeard.js` so as to distinguish it from the wiki Treebeard table of contents.